tikv-configuration-file.md

```diff
@@ -496,6 +502,12 @@ Configuration items related to Raftstore
 + Default value: 3/4 of the Region size
 + Minimum value: greater than `0`
 
+### `raft-log-reserve-max-ticks` <span class="version-mark">New in v5.3</span>
+
++ Old Raft logs could be reserved if `raft-log-gc-threshold` is not reached. GC them after `raft-log-reserve-max-ticks` ticks.
```

Comment:
```suggestion
+ If the maximum allowable count of residual Raft logs did not reach the value of `raft-log-gc-threshold`, old Raft logs are reserved. In this case, after `raft-log-reserve-max-ticks` is set, TiKV performs garbage collection (GC) to those Raft logs.
```

Reply:
Could you improve based on this new version?
"Performs garbage collection (GC) to residual Raft logs after `raft-log-reserve-max-ticks` ticks, even if their count doesn't reach the value of `raft-log-gc-threshold`."
And the "ticks" refers to elapsed time.